### ISSUE TITLE
Block certain IPs from tracking results

### ISSUE THREAD

**Issue by snipe** [jenssegers/laravel-ab#2]
It would be really helpful to be able to list specific IP addresses to prevent their results from being tracked (for example, the developers IPs). I may try to work on this if it's something you think you might want added here.

**Comment by daya-shankar** [comment]
@snipe I agree. I am willing to work on this too if you want this ability added.
EDIT: Forked the project and added this function. It's on my account.
